committee-steering/governance/sig-governance.md

```diff
@@ -90,28 +90,36 @@ Subproject contributors (as applicable).
 - *SHOULD* identify, track, and maintain the SIGs enhancements for current
   release and serve as point of contact for the release team, but *MAY* delegate
    to another Lead to fulfill these responsibilities
-  - *MAY* delegate the creation of a SIG roadmap to other Leads
-  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
-  including subprojects and their meeting information but *SHOULD* delegate the
-  need for subproject meetings to subproject owners  
-  - *SHOULD* facilitate meetings but *MAY* delegate to other Leads or future
-  chairs/chairs in training
+- *MUST* ensure the SIG roadmap is shared regularly. *MAY* delegate the creation
+  of a SIG roadmap to other Leads
+- *MUST* watch for the active issues and PRs trends
```

Comment:
This seems like an excessive requirement for Chairs.

Chairs help facilitate the organization of a SIG, such as meetings an annual reports, but do not have to be personally engaged in all subprojects.

Demanding that all chairs watch all possibly SIG related issues seems like a big requirement.

Some SIGs may have dozens of projects with individual area leads.

Reply:
2020 annul report had this question: https://github.com/search?q=repo%3Akubernetes%2Fcommunity+%22What%27s+the+average+open+days+of+a+PR+and+Issue+in+your+group%3F%22&type=code So watching trends is not something without precedence. Even if it's once a year.

Do you think it is not a responsibility or this is not chair responsibility?